pkg/apis/operator/v1alpha1/operandregistry_types.go

```diff
@@ -86,7 +86,7 @@ type OperandRegistryStatus struct {
 	// Phase describes the overall phase of operators in the OperandRegistry
 	// +operator-sdk:gen-csv:customresourcedefinitions.statusDescriptors=true
 	// +optional
-	Phase OperatorPhase `json:"phase,omitempty"`
+	Phase RegistryPhase `json:"phase,omitempty"`
```

Comment:
don't change the api types, let's keep it as it.

When we bump the new apiversion, we can do this, but still need to keep backward compatibility.

Reply:
It doesn't matter, API not change, `OperatorPhase` and `RegistryPhase` are both `string` type, so no impact.